app/views/referee_interface/reference/finish.html.erb

```diff
@@ -0,0 +1,9 @@
+<main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
+  <h1 class="govuk-heading-xl">Thank you</h1>
+    <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+    <% if @allows_user_research %>
+      <h2 class="govuk-heading-m">Our user research team will contact you shortly </h2>
+    <% end %>
+    <p class="govuk-body">If you have any questions about the Apply for teacher training service, please contact <a href="mailto:becomingateacher@digital.education.gov.uk">becomingateacher@digital.education.gov.uk</a>.</p></div>
```

Comment:
There's a helper you can use for the bat support email. This will also put the `govuk-link` class onto the link.

Reply:
Awesome, thanks! 👌